lib/assert.js

```diff
@@ -320,6 +320,18 @@ function wrapAssertions(callbacks) {
 				return;
 			}
 
+			try {
+				if (isPromise(fn().catch(_ => {}))) {
+					fail(this, new AssertionError({
+						assertion: 'throws',
+						improperUsage: true,
+						message: '`t.throws()` cannot be called with an asynchronous function. Use `t.throwsAsync()` instead',
+						values: [formatWithLabel('Called with:', fn)]
```

Comment:
I don't think printing the `fn` function is useful in this scenario. Let's remove `values` altogether.

Reply:
Thanks for the feedback! I can work on making these changes this week.